exercise/letter-space-number.js

```diff
@@ -0,0 +1,29 @@
+/*
+## Letter Space Number
+
+### Instruction
+
+Make a function called `letterSpaceNumber` that receives a string and returns an array with every instance of a letter, followed by a space, followed by a number only if that number has only one digit.
+
+### Notions
+
+- https://www.regular-expressions.info/shorthand.html
+
+// /*/ // ⚡
+
+
+
+// /*/ // ⚡
+
+export const tests = []
+const t = f => tests.push(f)
+
+t(({eq}) => eq(typeof letterSpaceNumber, "function")
+t(({eq}) => eq(letterSpaceNumber.length, 1))
+t(({eq}) =>
+  eq(letterSpaceNumber('He is 8 or 9 years old, not 10.'), ['s 8', 'r 9'])
+)
+t(({eq}) => eq(letterSpaceNumber("I like 7up."), []))
+t(({eq}) => eq(letterSpaceNumber("Definitely 9."), ["y 9"]))
```

Comment:
please apply prettier and ad more cases.
reading the subject it's not clear why `7up` is not matched.
what do you need to end the number bounds ?
why do `.` end the number and not `u` should `-` and `_` also end numbers ?
what about the end of the line ?
```js
'9 to 5' -> ['o 5']
```
Please add more tests and be more specific in the subject :)

Reply:
(good work, non the less, the idea is solid)